### PR TITLE
fix swagger yaml to match the code

### PR DESCRIPTION
**Pull request by yuval-k**
Description of changes:
Changes to the yaml to match the code better.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by yuval-k** [comment]
Good to know.
what about the fault message?
is fix planned in the code, or should I update this PR to only contain that?

File: api_server/swagger/firecracker.yaml [hunk 1]
```diff
@@ -360,7 +362,7 @@ definitions:
 
   Error:
     properties:
-      faultMessage:
+      fault_message:
```
Comment (acatangiu) [inline api_server/swagger/firecracker.yaml]: This change is ok.

File: api_server/swagger/firecracker.yaml [hunk 2]
```diff
@@ -224,8 +224,10 @@ paths:
           schema:
             type: object
       responses:
+        201:
+          description: MMDS data store created
         204:
-          description: MMDS data store created/updated.
```
Comment (acatangiu) [inline api_server/swagger/firecracker.yaml]: This doesn't need to change.

**Comment by yuval-k** [comment]
forced push only the relevant change

**Pull request merged**